Cargo.toml

```diff
@@ -53,8 +55,9 @@ hex = { version = "0.4", features = ["serde"] }
 [features]
 std = []
 virtual = ["std", "vpicc"]
-rsa2048 = ["trussed/rsa2048"]
-rsa4096 = ["rsa2048", "trussed/rsa4096"]
+rsa = ["trussed-rsa-backend"]
+rsa2048 = ["rsa"]
```

Comment:
Do we still need this as a separate feature?

Reply:
It still requires an allocator AKA nightly. It also doesn't solve the problem of slow key generation which means we still need the feature flag for RSA 4096 key generation. I also need to test how long it takes for RSA-3072